src/sentry/testutils/cases.py

```diff
@@ -745,7 +745,7 @@ def setUp(self):
         self.save_session()
 
     def assertDialogSuccess(self, resp):
-        assert "window.opener.postMessage(" in resp.content
+        assert b"window.opener.postMessage(" in resp.content
```

Comment:
Just checking that it makes sense for this to be bytes here.

Reply:
https://github.com/getsentry/sentry/pull/17382#discussion_r385986979